manifests/init.pp

```diff
@@ -72,6 +72,14 @@
 #                                  (30 or 30s), minutes (30m), hours (6h), days (2d),
 #                                  or years (5y).
 #
+# $autosign::                      Eanble or disable autosign or change location
+#                                  of autosign.conf or autosign script.  If this
+#                                  is set to a script, make sure that script
+#                                  considers the content of autosign.conf, as it
+#                                  might break Foreman functionality if it
+#                                  doesn't.
+#                                  type:string or boolean
```

Comment:
this doesn't work with kafo:
`/usr/lib/ruby/vendor_ruby/kafo/param_builder.rb:83:in 'const_defined?': wrong constant name String or boolean (NameError)`
@ares what's the best to do in such a case?